crates/db_connection_pool/Cargo.toml

```diff
@@ -34,9 +34,9 @@ odbc-api = { workspace = true, optional = true }
 arrow-odbc = { workspace = true, optional = true }
 lazy_static = "1.4.0"
 clickhouse-rs = { workspace = true, optional = true }
+tokio-postgres = "0.7.10"
```

Comment:
We alreaady have a dependency for it in workspace

```suggestion
tokio-postgres = { workspace = true }
```

Reply:
This should also be optional and enabled by the `postgres` feature

Reply:
updated to optional tokio postures dependency